mdbenchmark/analyze.py

```diff
@@ -61,22 +61,7 @@ def analyze(directory, plot, ncores, output_name):
     """Analyze finished benchmarks."""
     bundle = mds.discover(directory)
 
-    df = pd.DataFrame(
-        columns=["module", "nodes", "ns/day", "run time [min]", "gpu", "host", "ncores"]
-    )
-
-    for i, sim in enumerate(bundle):
-        # older versions wrote a version category. This ensures backwards compatibility
-        if "module" in sim.categories:
-            module = sim.categories["module"]
-        else:
-            module = sim.categories["version"]
-        # call the engine specific analysis functions
-        engine = detect_md_engine(module)
-        df.loc[i] = utils.analyze_run(engine=engine, sim=sim)
-
-    if df.empty:
-        console.error("There is no data for the given path.")
+    df = DataFrameFromBundle(bundle)
```

Comment:
`DataFrameFromBundle` is not imported and therefore `mdbenchmark analyze` is not working.